pgmpy/independencies/Independencies.py

```diff
@@ -50,9 +50,16 @@ def __str__(self):
 
     __repr__ = __str__
 
+    def __eq__(self, other):
+        for self_independency in self.get_assertions():
+            if self_independency not in other.get_assertions():
```

Comment:
@kshitij10496 You can avoid calling `other.get_assertions()` everytime in the loop. Better call it once before the loop and store it's value in some variable.